src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  border: none;
+  background: none;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.cover {
+  margin: 20px 8px 8px;
+}
+
+.form {
+  position: relative;
+  display: block;
+  width: auto;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  height: 70px;
+}
+
+.smallForm {
+  height: 42px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  border: none;
+  border-radius: 4px;
+  outline: none;
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
```

Comment:
```suggestion
  font-family: inherit;
```

Reply:
@liliia-livandovskaya Thank you! Made changes